pages/home/_meta.json

```diff
@@ -5,6 +5,10 @@
   },
   "what-is-safe": "What is Safe?",
   "safe-core": "Safe{Core}",
+  "tutorials": {
+    "title": "Tutorials",
+    "href": "/resource-hub?source=Safe+Team&tag=Tutorial"
```

Comment:
No "Blog Post" tag as well in the end?

Reply:
Yeah, I ended up skipping it as I realized that we would also want to list @valle-xyz's tutorial videos here in the future.

Reply:
I like it as simple as that. The two filters are enough to show only company tutorials.